csrfguard/src/main/resources/csrfguard.properties

```diff
@@ -334,29 +334,34 @@ org.owasp.csrfguard.Config.Print = true
 # If it's left blank, and it's not configured in the web.xml either, it defaults to META-INF/csrfguard.min.js.
 # Use of this property and the existence of the specified template file is required.
-org.owasp.csrfguard.JavascriptServlet.sourceFile = 
+org.owasp.csrfguard.JavascriptServlet.sourceFile =
 
 # Boolean value that determines whether or not the dynamic JavaScript code should be strict
-# with regards to what links it should inject the CSRF prevention token. With a value of true, 
-# the JavaScript code will only place the token in links that point to the same exact domain 
-# from which the HTML originated. With a value of false, the JavaScript code will place the 
-# token in links that not only point to the same exact domain from which the HTML originated, 
+# with regards to what links it should inject the CSRF prevention token. With a value of true,
+# the JavaScript code will only place the token in links that point to the same exact domain
+# from which the HTML originated. With a value of false, the JavaScript code will place the
+# token in links that not only point to the same exact domain from which the HTML originated,
 # but sub-domains as well.
 org.owasp.csrfguard.JavascriptServlet.domainStrict = true
 
-# Allows the developer to specify the value of the Cache-Control header in the HTTP response 
+# Allows the developer to specify the value of the Cache-Control header in the HTTP response
 # when serving the dynamic JavaScript file. The default value is private, max-age=28800.
-# Caching of the dynamic JavaScript file is intended to minimize traffic and improve performance. 
-# Note that the Cache-Control header is always set to "no-store" when either the "Rotate" 
+# Caching of the dynamic JavaScript file is intended to minimize traffic and improve performance.
+# Note that the Cache-Control header is always set to "no-store" when either the "Rotate"
 # "TokenPerPage" options is set to true in Owasp.CsrfGuard.properties.
 org.owasp.csrfguard.JavascriptServlet.cacheControl = private, max-age=28800
 
-# Allows the developer to specify a regular expression describing the required value of the 
-# Referer header. Any attempts to access the servlet with a Referer header that does not 
-# match the captured expression is discarded. Inclusion of referer header checking is to 
-# help minimize the risk of JavaScript Hijacking attacks that attempt to steal tokens from 
-# the dynamically generated JavaScript. While the primary defenses against JavaScript 
-# Hijacking attacks are implemented within the dynamic JavaScript itself, referer header 
+# Allows the developer to specify the value of the Cache-Control header in the HTTP response
+# when serving the dynamic JavaScript file included with a tag query param that equals the generated ETag.
+# In that specific case you can assume a client cache without revalidation and use a custom cache duration.
+org.owasp.csrfguard.JavascriptServlet.cacheControlTagged = private, max-age=600
```

Comment:
Please mention it in the documentation that it should not be used together with configuration options like token rotation, because it will only cause an overhead.